test/preprocessors/BasePreprocessor.test.ts

```diff
@@ -64,64 +73,61 @@ describe("BasePreprocessor", () => {
   });
 
   describe("#preprocess()", () => {
-    beforeEach(() => {
-      const mockBlacklistPredicate: Predicate<EntityDefinition> = {
-        evaluate: mockEvaluate,
-      };
-
-      const mockResourceTransformer: Transformer<
-        EntityDefinition,
-        EntityDefinition
-      > = {
-        transform: mockResourceTransform,
-      };
-
-      const mockElementTransformer: Transformer<
-        EntityDefinition,
-        EntityDefinition
-      > = {
-        transform: mockElementTransform,
-      };
-
-      preprocessor.blacklistPredicate = mockBlacklistPredicate;
-      preprocessor.addResourceTypeTransformer = mockResourceTransformer;
-      preprocessor.modifyElementTypeTransformer = mockElementTransformer;
-    });
-
     it("should filter out blacklisted EntityDefinitions", () => {
       const result = preprocessor.preprocess(entityCollection);
       expect(result).not.toBe(entityCollection);
-      expect(result.entities).toBeArrayOfSize(3);
-      expect(mockEvaluate).toHaveBeenCalledTimes(2);
+      expect(result.entities).toBeArrayOfSize(4);
+      expect(result.entities[0].dataType.normalizedName).toBe("Resource");
+      expect(result.entities[1].dataType.normalizedName).toBe("Element");
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5963561)